blocks/header-nav-chain-block/chains/header-nav-chain-block/_children/nav-widget.jsx

```diff
@@ -22,10 +22,11 @@ const NavWidget = ({
   const predefinedWidget = (
     (type === 'search' && (
       <SearchBox
-        iconSize={ICON_SIZE}
+        iconSize={WIDGET_CONFIG[placement].iconSize}
```

Comment:
@vinnie-olsen would it make sense to use optional chaining here and on line 29?

Reply:
@beltrancaliz yep that makes sense - I'll add that change 👍  It's probably not 100% crucial here since that `iconSize` should _always_ exist for each entry in the `WIDGET_CONFIG` but probably good to have either way.